dtm0/drlink.h

```diff
@@ -0,0 +1,81 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+
+#pragma once
+
+#ifndef __MOTR_DTM0_DRLINK_H__
+#define __MOTR_DTM0_DRLINK_H__
+
+/*
+ * DTM0 RPC Link is a lazy ("on-demand") RPC link.
+ *
+ * Whenever ::m0_dtm0_req_post is called, it launches a FOM that is trying
+ * to connect to the target process (if there is no connection) and then
```

Comment:
What if the rpc connect is already establish?

Reply:
If there is no connection then it is not trying to connect to the target process (as per the note in the brackets). So, it does nothing if the connection has already been established.